patches/rawspeed-ub.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/librawspeed/decompressors/VC5Decompressor.cpp b/src/librawspeed/decompressors/VC5Decompressor.cpp
+index 54e5145d..be9fcee3 100644
+--- a/src/librawspeed/decompressors/VC5Decompressor.cpp
++++ b/src/librawspeed/decompressors/VC5Decompressor.cpp
+@@ -790,6 +790,9 @@ void VC5Decompressor::parseLargeCodeblock(ByteStream bs) {
+   } else {
+     if (!mVC5.quantization.has_value())
+       ThrowRDE("Did not see VC5Tag::Quantization yet");
++    if (!codeDecoder.has_value()) {
++      ThrowRDE("You did hit bug https://github.com/darktable-org/darktable/issues/15022");
++    }
```

Comment:
whether it's intentional or not, it's UB. So it's a bug in the code. 
As for performance, I'd take correctness over performance.

BTW it does trigger on every GPR files I have, so it's not a fluke. That mean it would assert on every one of them.